### PR TITLE
update model description

### PR DESCRIPTION
**Pull request by soulseen**
Signed-off-by: soulseen <sunzhu@yunify.com>

### CODE REVIEW

File: pkg/models/devops/json.go [hunk 1]
```diff
@@ -1078,17 +1078,17 @@ type NodesDetail struct {
...
+	Actions            []interface{} `json:"actions,omitempty" description:"the list of all actions"`
 	DisplayDescription interface{}   `json:"displayDescription,omitempty" description:"display description"`
 	DisplayName        string        `json:"displayName,omitempty" description:"display name"`
 	DurationInMillis   int           `json:"durationInMillis,omitempty" description:"duration time in millis"`
 	ID                 string        `json:"id,omitempty" description:"id"`
-	Input              interface{}   `json:"input,omitempty" description:"input"`
-	Result             string        `json:"result,omitempty" description:"result"`
-	StartTime          string        `json:"startTime,omitempty" description:"start time"`
-	State              string        `json:"state,omitempty" description:"statue"`
+	Input              interface{}   `json:"input,omitempty" description:"the data of user input"`
```
Comment (runzexia) [inline pkg/models/devops/json.go]: input step?

File: pkg/models/devops/json.go [hunk 2]
```diff
@@ -1006,9 +1006,9 @@ type NodeSteps struct {
 	DisplayName        string      `json:"displayName,omitempty" description:"display name"`
 	DurationInMillis   int         `json:"durationInMillis,omitempty" description:"duration time in mullis"`
 	ID                 string      `json:"id,omitempty" description:"id"`
-	Input              interface{} `json:"input,omitempty" description:"input"`
-	Result             string      `json:"result,omitempty" description:"result"`
-	StartTime          string      `json:"startTime,omitempty" description:"start times"`
+	Input              interface{} `json:"input,omitempty" description:"the data of user input"`
```
Comment (runzexia) [inline pkg/models/devops/json.go]: input step?

File: pkg/models/devops/json.go (hunk 1 above)
Comment (soulseen) [reply pkg/models/devops/json.go]: yes.

File: pkg/models/devops/json.go (hunk 2 above)
Comment (soulseen) [reply pkg/models/devops/json.go]: yes.

**Pull request closed without merging**